packages/react-native/ReactCommon/react/renderer/attributedstring/AttributedString.cpp

```diff
@@ -91,6 +91,18 @@ void AttributedString::prependAttributedString(
       attributedString.fragments_.end());
 }
 
+int AttributedString::countAttachments() const {
+  int count = 0;
+
+  for (const auto& fragment : fragments_) {
+    if (fragment.isAttachment()) {
+      ++count;
```

Comment:
nit: note that the other code is `count++` rather than `++count`. I know that in this context it does not make a difference, but I'd rather keep the same semantic if we don't need to change it.